homeassistant/components/nfandroidtv/config_flow.py

```diff
@@ -29,8 +55,8 @@ async def async_step_user(
             host = user_input[CONF_HOST]
             name = user_input[CONF_NAME]
 
+            self._async_abort_entries_match({CONF_HOST: host})
             await self.async_set_unique_id(host)
```

Comment:
This doesn't look like an acceptable unique id 
https://developers.home-assistant.io/docs/config_entries_config_flow_handler?_highlight=unique&_highlight=id&_highlight=re#unique-id-requirements

Reply:
Yes, it is but that's why we have DHCP discovery check for existing entries. If one is found, the unique id changes to the mac address. The mac address exists in layer 2 so we only see it via DHCP. If the user tries to add an entry, it aborts even though the mac address obviously never matches the ip.

There is no better unique id available for the user step. Non-internal integrations are not currently allowed to depend on others at all. So I can't compare an entry from say `androidtv`.

Reply:
We shouldn't see the unique id to `host` though. Its better to not have one than to have an unacceptable one